src/ert/cli/main.py

```diff
@@ -130,12 +129,12 @@ def run_cli(args: Namespace, _: Any = None) -> None:
         else:
             out = sys.stderr
         monitor = Monitor(out=out, color_always=args.color_always)
-
+        thread.start()
         try:
-            monitor.monitor(tracker.track())
-        except (SystemExit, KeyboardInterrupt):
+            monitor.monitor(status_queue)
+        except (SystemExit, KeyboardInterrupt, OSError):
             print("\nKilling simulations...")
-            tracker.request_termination()
+            model.cancel()
```

Comment:
Also need to stop the monitor here?

Reply:
We can make it explicit that this communicates over a queue on the outside, but I think the next step here is the simplify the entire run model API anyway?